Documentation/ApiOverview/FlexForms/Index.rst

```diff
@@ -0,0 +1,380 @@
+                    </numIndex>
+                    <numIndex index="1">
+                        <numIndex index="0">LLL:EXT:example/Resources/Private/Language/Backend.xlf:settings.registration.orderBy.tstamp</numIndex>
+                        <numIndex index="1">tstamp</numIndex>
+                    </numIndex>
+                    <numIndex index="2">
+                        <numIndex index="0">LLL:EXT:example/Resources/Private/Language/Backend.xlf:settings.registration.orderBy.datetime</numIndex>
+                        <numIndex index="1">datetime</numIndex>
+                    </numIndex>
+                    <numIndex index="3">
+                        <numIndex index="0">LLL:EXT:example/Resources/Private/Language/Backend.xlf:settings.registration.orderBy.crdate</numIndex>
+                        <numIndex index="1">crdate</numIndex>
+                    </numIndex>
+                    <numIndex index="4">
+                        <numIndex index="0">LLL:EXT:example/Resources/Private/Language/Backend.xlf:settings.registration.orderBy.title</numIndex>
+                        <numIndex index="1">title</numIndex>
+                    </numIndex>
+                </items>
+            </config>
+        </TCEforms>
+    </settings.orderBy>
+
+.. _flexforms-itemsProcFunc:
+
+Populate a `select` Field with a PHP Function (itemsProcFunc)
+-------------------------------------------------------------
+
+Here, the
+
+* vendor name is **Myvendor**
+* the extension key is **example**
+
+.. code-block:: xml
+
+    <settings.orderBy>
+        <TCEforms>
+            <label>LLL:EXT:example/Resources/Private/Language/Backend.xlf:settings.registration.orderBy</label>
+            <config>
+                <type>select</type>
+                <itemsProcFunc>Myvendor\Example\Backend\ItemsProcFunc->user_orderBy</itemsProcFunc>
+                <renderType>selectSingle</renderType>
+                <items>
+                    <!-- empty by default -->
+                </items>
+            </config>
+        </TCEforms>
+    </settings.orderBy>
+
+The function :php:`user_orderBy` populates the select field in
+:file:`Backend\ItemsProcFunc.php`:
+
+.. code-block:: php
+
+    class ItemsProcFunc
+    {
+         /**
+         * Modifies the select box of orderBy-options.
+         *
+         * @param array &$config configuration array
+         */
+        public function user_orderBy(array &$config)
+        {
+            // simple and stupid example
+            // change this to dynamically populate the list!
+            $config['items] = [
+                // label, value
+                ['Timestamp', 'timestamp'],
+                ['Title', title]
+            ];
+        }
+
+        // ...
+     }
+
+How this looks when configuring the plugin:
+
+.. image:: Images/FlexFormsItemsProcFunc.png
+   :class: with-shadow
+
+Select a Controller Action
+==========================
+
+.. code-block:: xml
```

Comment:
There is no explanation what this example is about. Maybe we should add at least one sentence or paragraph.